travel_platform/test_full_landing/travel_platform.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
```

Comment:
the code from travel_platform is divided between base and main. There are django tags added to these two files.